### PR TITLE
feature/CI

### PR DESCRIPTION
**Pull request by DaniilZ77**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -54,20 +55,7 @@ jobs:
...
-        - name: Checkout repository
-          uses: actions/checkout@v4
-
-        - name: Setup Go
-          uses: actions/setup-go@v5
-          with:
-            go-version: '1.22.5'
-        
+          
         - name: Install golangci-lint
```
Comment (petrukhinandrew) [inline .github/workflows/ci.yml]: not sure if it is better (at least, seems more verbose) and if it actually works, but may be use https://github.com/golangci/golangci-lint-action instead?

**Review by petrukhinandrew — COMMENTED** [review]
seems awesome

File: .github/workflows/ci.yml (hunk 1 above)
Comment (DaniilZ77) [reply .github/workflows/ci.yml]: changed to golangci-lint-action

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -54,25 +55,9 @@ jobs:
...
-          run: |
-            curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.61.0
-
+          
         - name: Lint
-          run: |
-            cd backend
-            make lint
\ No newline at end of file
+          uses: golangci/golangci-lint-action@v6
```
Comment (petrukhinandrew) [inline .github/workflows/ci.yml:60]: I'm excited

**Review by petrukhinandrew — APPROVED** [review]
awesome

**Pull request merged**